db_test.go

```diff
@@ -1035,6 +1035,17 @@ func TestWithPerconaDeadline(t *testing.T) {
 		!strings.HasSuffix(logger.lastQuery, " (SELECT * FROM objects ORDER BY ID ASC LIMIT 1) UNION (SELECT * FROM objects ORDER BY id DESC LIMIT 1)") {
 		t.Fatalf("Expected %q, got %q", "SET STATEMENT max_statement_time=? FOR (SELECT * FROM objects ORDER BY ID ASC LIMIT 1) UNION (SELECT * FROM objects ORDER BY id DESC LIMIT 1)", logger.lastQuery)
 	}
+
+	// Test white space
+	logger.lastQuery = ""
+
+	db.QueryRow(`
+     SELECT * from objects LIMIT 1`)
+
+	if !strings.HasPrefix(logger.lastQuery, "SET STATEMENT max_statement_time=") ||
```

Comment:
This iis checking for the Percona syntax, while the fix you made is in the MySQL 5.7 rewriter.